### PR TITLE
docs(legal): update cookies in privacy policy

### PR DESCRIPTION
**Pull request by mffap**
We will introduce google analytics to improve marketing and user experience analytics on our websites. Analytics is only done on our public website and the customer portal. We don't track behavior with Google Analytics on customer instances.

- [ ] Update date

### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/legal/policies/privacy-policy.mdx [hunk 1]
```diff
@@ -117,25 +117,34 @@ Our websites use cookies. These are small text files that make it possible to st
...
+We cannot provide you with our services without this type of cookies.
 
-\_\_state  
-\_\_pkce.code  
-\_\_callback-url  
-
-Cookies are only used for technical purposes to enable the functionality and efficient use of our website and our offers, such as:
+Necessary cookies provide basic functionality such as:
 
 - Session management
```
Comment (fforootd) [inline docs/docs/legal/policies/privacy-policy.mdx:130]:
```suggestion
- Session Management
```

File: docs/docs/legal/policies/privacy-policy.mdx [hunk 2]
```diff
@@ -117,25 +117,34 @@ Our websites use cookies. These are small text files that make it possible to st
...
 - Single Sign On
```
Comment (fforootd) [inline docs/docs/legal/policies/privacy-policy.mdx:131]:
```suggestion
- Single Sign-On
```

File: docs/docs/legal/policies/privacy-policy.mdx [hunk 3]
```diff
@@ -117,25 +117,34 @@ Our websites use cookies. These are small text files that make it possible to st
...
 - Rate Limiting
 - DDoS Mitigation
+- Remembering preferences
```
Comment (fforootd) [inline docs/docs/legal/policies/privacy-policy.mdx:134]:
```suggestion
- Remembering Preferences
```

File: docs/docs/legal/policies/privacy-policy.mdx [hunk 4]
```diff
@@ -117,25 +117,34 @@ Our websites use cookies. These are small text files that make it possible to st
...
+
+### Analytical cookies
+
+We also use cookies for website analytics purposes in order to operate, maintain, and improve the services for you.
+We use Google Analytics 4 to collect and process certain analytics data on our behalf.
+Google Analytics helps us understand how you engage with the services and may also collect information about your use of other websites, apps, and online resources.
+We don't use google analytics on customer instances of ZITADEL, but only on our public websites and customer portal.
```
Comment (fforootd) [inline docs/docs/legal/policies/privacy-policy.mdx:141]:
```suggestion
We don't use google analytics on customer instances of ZITADEL, only on our public websites and customer portal.
```

**Review by fforootd — COMMENTED** [review]
some small recommendations

File: docs/docs/legal/policies/privacy-policy.mdx [hunk 5]
```diff
@@ -4,7 +4,7 @@ custom_edit_url: null
 --- 
 import PiidTable from '../_piid-table.mdx';
 
-Last updated on November 15, 2023
+Last updated on March 01, 2024
```
Comment (eliobischof) [inline docs/docs/legal/policies/privacy-policy.mdx:7]:
```suggestion
Last updated on March 07, 2024
```

**Review by eliobischof — CHANGES_REQUESTED** [review]

**Review by eliobischof — APPROVED** [review]

**Pull request merged**